cylc/flow/config.py

```diff
@@ -786,7 +786,6 @@ def _check_implicit_tasks(self) -> None:
             not cylc.flow.flags.cylc7_back_compat
         ):
             raise WorkflowConfigError(msg)
-        LOG.warning(msg)
```

Comment:
This log message says "to allow implicit tasks ..." which isn't appropriate in Cylc 7 compat mode (as implicit tasks are already allowed by exception), if users followed this advice they would make the workflow Cylc 7 incompatible.